### PR TITLE
Feature file feedback

### PR DESCRIPTION
**Pull request by timothyryanwalsh**
Hi, all! At CCA's request, I took a look at the feature files and had a few suggestions for changes. Overall, they are in really great shape! I'm very happy to see this project progressing so well and in great hands.

Some of the suggestions included here reflect changes (e.g. the addition of a home page separate from the Collections page; and adding the Collection to the All Digital Files data table) that were implemented by Bun in his UI prototype following user testing, but seemed to be missing in the feature files. It might be worth another look at his UXPin prototype (I don't have access to it) to make sure all of the changes are accounted for here.

The changes in how the async background processes are communicated to the user in the UI are just suggestions - take or leave them, but I think it might be less confusing to a user than being dropped back in the home page after uploading something.

Looking forward to discussing further with you all on Wednesday.

Connects to #34.

### CODE REVIEW

File: features/data_tables.feature [hunk 1]
```diff
@@ -59,29 +60,33 @@ Feature: Data tables
...
     And the table header will show sort options for the following columns
-      | column_name  |
-      | Filepath     |
-      | Format       |
-      | Size (bytes) |
+      | column_name   |
+      | Filepath      |
+      | Format        |
+      | Size (bytes)  |
+      | Last modified |
```
Comment (jraddaoui) [inline features/data_tables.feature]: This field is currently being stored as a string field. We can do the same as we do with the identifier and title fields to allow sorting over it (indexing it two times in ES with different field types), but ideally it should be stored in the DB and the ES index as a date time field.

I've created #54 to take care of that in the future.

File: features/dublin_core_settings.feature [hunk 2]
```diff
@@ -29,12 +29,13 @@ Feature: Dublin Core settings
         | language    |
         | coverage    |
         | rights      |
+        | isPartOf    |
```
Comment (jraddaoui) [inline features/dublin_core_settings.feature]: Created #55 to add this field to the DublinCore model.

File: features/user_management_and_permissions.feature [hunk 3]
```diff
@@ -46,7 +46,7 @@ Feature: User management and permissions
     And the viewer can browse and view Folders
     And the viewer can browse and view Digital Files
     And the viewer can download the Folders ZIP files
-    And the viewer cannot manage users
+    And the viewer cannot manage users except their own user account
```
Comment (jraddaoui) [inline features/user_management_and_permissions.feature]: Created #56 to allow users to edit their own account.

**Comment by timothyryanwalsh** [comment]
Thanks @stefanabreitwieser! I that was Radda or Kelly's idea - I actually thought it might be better to restrict that to Administrators. It seemed to me that which fields to display would likely be a policy at an institution-level, so it would make sense to set it once and forget it. My thought was that otherwise, it might get confusing as different users see different elements in the interface, but maybe we should talk about it more tomorrow.

**Comment by jraddaoui** [comment]
Hi @timothyryanwalsh, @stefanabreitwieser,

We thought the settings will be global settings for the instance, not specific to each user. The question in the feature file was if Editors and/or Managers should be able to edit those global settings.

Looking forward for tomorrow's meeting.

**Comment by jraddaoui** [comment]
Hi @timothyryanwalsh, please, could you squash the commits in one so we can merge with `--ff-only` to maintain a clean commit history.

Thanks!

**Comment by timothyryanwalsh** [comment]
Hi @jraddaoui, that should do it!

**Pull request merged**